src/components/CardEditor.tsx

```diff
@@ -61,26 +49,35 @@ export default function CardEditor(props: Props) {
         program.create_decks.length !== 0 &&
         program.create_decks[program.create_decks.length - 1].deck === null
       ) {
-        console.log("last deck is null, not sending dispatch");
+        debug("last deck is null, not sending dispatch");
       } else {
-        localStorage.setItem("programAST", JSON.stringify(program));
+        localStorage.setItem(astStrKey, JSON.stringify(program));
+        // Trigger background reconciliation with DB
+        reconcile(props.program, program, db)
+          .then(() => debugDB("Background DB update complete!"))
+          .catch((err) => debugDB("Error during reconciliation!", err));
```

Comment:
Background DB update call, taking the previous and current `PROGRAM`s (`props.dispatch` then places the new `program` as the `props.program`)